### ISSUE TITLE
main page scrollbar completely invisible in chrome

### ISSUE THREAD

**Issue by lovasoa** [tabler/tabler#1648]
**Describe the bug**

Page scrollbars are not visible in google chrome.

Initially reported in https://github.com/lovasoa/SQLpage/discussions/17

**Device**

Google Chrome | 114.0.5735.198 (Official Build) (64-bit)
-- | --
Revision | c3029382d11c5f499e4fc317353a43d411a5ce1c-refs/branch-heads/5735@{#1394}
OS | Linux
JavaScript | V8 11.4.183.25

**To reproduce**

1. Go to https://sql.ophir.dev/ using google chrome
2. Look at the right of the page: you don't see the page's scrollbar, even through the page is scrollable. There is space allocated to the scrollbar, but it is not visible.

**Screenshots**

![image](https://github.com/tabler/tabler/assets/552629/cff62d4f-ea77-4287-9dd5-99386fd09e3a)

you can see the scrollbar is here, it takes up space, but is not visible:
![image](https://github.com/tabler/tabler/assets/552629/1c2c7a3d-822d-4d21-9116-e015544032e7)

If you manually remove tabler's scrollbar styling, the issue disappears:
![image](https://github.com/tabler/tabler/assets/552629/4aaddab8-908c-478c-a551-8cddfd0e5c7e)

**Comment by rjd22** [comment]
This seem not to be a bug but an implementation conflict. The background color you chose seems to hide the scollbar. If you remove the background color you will be able to find the styled scrollbar. At least on my machine this seems to be the case.

In this case if you want to used the unstyled scrollbar I would recommend overruling the scrollbar rules for your specific implementation.

**Comment by rjd22** [comment]
You're right. It seems to have more to so with that we're touching `scrollbar-color` in this location to make it more soft:

https://github.com/tabler/tabler/blob/aba771f23d4f1583fb9b1b85840dfed0ec81cbda/src/scss/mixins/_mixins.scss#L18

You could for now overrule this with a more obvious color to fix it for your situation for now. @codecalm might it be an idea to remove this mixin completely and let the browser decide the scrollbar color? IMO we should not mess with user interface colors to much to avoid situations like these.

**Comment by LeBogoo** [comment]
I also ran into this today. I'm only using `data-bs-theme="dark"` on the body tag and the `Sidebar layout` example from the docs.

Is there any fix planned for the near future?

**Comment by itopaloglu83** [comment]
For some reason the scrollbar is forced to be always visible in Safari. Default macOS settings hide the scrollbar when not in motion, so this creates an unexpected visual cue for the users. 

```CSS
::-webkit-scrollbar {
    width: 1rem;
    height: 1rem;
    -webkit-transition: background .3s;
    transition:background .3s
}
```

<img width="1728" alt="Image" src="https://github.com/user-attachments/assets/36017097-29c1-4950-b67e-68cec8271401" />

**Comment by ethancrawford** [comment]
@codecalm - does #2237 need to be reworked given @itopaloglu83's comment above? I can see the same scrollbar in Safari as in the screenshot as well. It appears that using `100vw` is causing further issues as described in https://github.com/tabler/tabler/issues/2271 also.
